### PR TITLE
Updated Contacts and Best Faculty Awards Page

### PR DESCRIPTION
**Pull request by MMuzzammil**
Please review and suggest changes, if any.

### CODE REVIEW

File: contact/index.html [hunk 1]
```diff
@@ -20,14 +20,12 @@
...
-    <link rel="preload" href="https://aca-cse-iitk.github.io/images/1.jpg" as="image" />
-    <link rel="preload" href="https://aca-cse-iitk.github.io/images/bhaskar.jpg" as="image" />
-    <link rel="preload" href="https://aca-cse-iitk.github.io/images/manish.jpg" as="image" />
-    <link rel="preload" href="https://aca-cse-iitk.github.io/images/4.jpg" as="image" />
+    <link rel="preload" href="https://aca-cse-iitk.github.io/images/apoorva.jpg" as="image" />
+    <link rel="preload" href="https://aca-cse-iitk.github.io/images/arpit.jpg" as="image" />
+    <link rel="preload" href="https://aca-cse-iitk.github.io/images/avideep.jpg" as="image" />
+    <link rel="preload" href="https://aca-cse-iitk.github.io/images/dev.jpg" as="image" />
+    <link rel="preload" href="https://aca-cse-iitk.github.io/images/rythm.jpg" as="image" />
+    <link rel="preload" href="https://aca-cse-iitk.github.io/images/rajat.jpg" as="image" />
```
Comment (dev-chauhan) [inline contact/index.html:28]: Add images in `images/` directory

Comment (MMuzzammil) [reply contact/index.html:28]: done.

**Comment by dev-chauhan** [comment]
Some images are distorted on the webpage. Crop them accordingly or adjust image size to remove distortion.

**Comment by MMuzzammil** [comment]
done.

**Pull request merged**